sdk/appconfiguration/azure-appconfiguration-provider/azure/appconfiguration/provider/_client_manager.py

```diff
@@ -135,11 +135,11 @@ def _check_configuration_setting(
 
     @distributed_trace
     def load_configuration_settings(
-        self, setting_selectors: List[SettingSelector], refresh_on_keys: Dict[Tuple[str, str], str], **kwargs
+        self, selects: List[SettingSelector], refresh_on: Mapping[Tuple[str, str], Optional[str]], **kwargs
```

Comment:
I see we previously renamed this to `setting_selectors`, is the intention to revert this change?

Reply:
This rename was the start of going down the wrong path of trying to fix an issue. If you look at `_azureappconfigurationproviderbase` it now has a method `sdk_allowed_kwargs` this is the suggested solution to this issue from the SDK team. TLDR, `kwargs` has a key in in named selects that was colliding with the property here.